### ISSUE TITLE
Setup quality of remote desktop

### ISSUE THREAD

**Issue by GusevVictor** [Ylianst/MeshCentral#818]
Dear all is it possible to setup 70-80-90-100% quality of remote desktop?

![изображение](https://user-images.githubusercontent.com/7081357/72223981-eceb7d80-3585-11ea-8924-2b9d52fffe60.png)

Best regards and thank for good software!

**Comment by Ylianst** [comment]
In the config.json file in meshcentral-data, add "AllowHighQualityDesktop": true. This will enable the extra modes. Now that people have requested this a bunch, I think I will switch the default to true in the future, only disallow these modes when sent to false.

```
{
  "settings": {
    "Port": 443,
    "AllowHighQualityDesktop": true
  }
}
```

**Comment by GusevVictor** [comment]
it works!

**Comment by Ylianst** [comment]
Just published MeshCentral v0.4.7-j where the default is to enable high quality modes. You need to set:

`"AllowHighQualityDesktop": false`

To disable these modes. Hope this closes this.

**Comment by GusevVictor** [comment]
Works
